scripts/cat/enums/status.py

```diff
@@ -0,0 +1,157 @@
+from scripts.game_structure.extended_strenum import ExtendedStrEnum
+
+
+class StatusEnum(ExtendedStrEnum):
```

Comment:
I know you specified a reason somewhere, but I personally find it a bit redundant to name variables, type definitions […] with their respective type. I know we're in Python and this is still a class rather than an actual enum, but calling a status enum "StatusEnum" is inconsistent with what we (should) usually do. I mean, think of the Cat class being called "CatClass"-

Reply:
I renamed them to this because I think I'd like to reorganise the Cat class to have an organisation object that holds status and a few other things. Enums can't hold other objects aside from Enums, so I was sort of future-proofing here to leave the `Status` namespace available. I can of course revert this and reconsider how I'd like to look at that organisation haha

Reply:
Wouldn't this be remedied if you imported the enums module and referred explicitly to the Status enum via `enums.Status`?

Reply:
I suppose it would, yeah. More than happy to go and do that if preferred. Speaking of, is the location of that module good in the file structure? I placed it within `cat` but I think it might make more sense for it to be further out, as there are other things that could want to use ExtendedStrEnum that may not be cat-related.

Reply:
Change made :)